crates/orchestrator/src/config.rs

```diff
@@ -182,7 +184,8 @@ impl<K: SignatureKey, E: ElectionConfig> NetworkConfig<K, E> {
     ///
     /// # Examples
     ///
-    /// ```no_run
+    /// ```ignore
+    /// use hotshot_orchestrator::config::NetworkConfig;
```

Comment:
Nit: aren't `use`-statements usually left out of examples?

Reply:
I would love to avoid the use statement, but I'm not sure how else to tell rust about type. If I drop that line (and retain `no_run` instead of `ignore`), the first error I get is:
```
failures:

---- crates/orchestrator/src/config.rs - config::NetworkConfig<K,E>::from_file (line 187) stdout ----
error[E0433]: failed to resolve: use of undeclared type `NetworkConfig`
 --> crates/orchestrator/src/config.rs:189:14
  |
4 | let config = NetworkConfig::from_file(file).unwrap();
  |              ^^^^^^^^^^^^^ use of undeclared type `NetworkConfig`
  |
help: consider importing this struct
  |
2 + use hotshot_orchestrator::config::NetworkConfig;
  |

error: aborting due to previous error
```

Reply:
It's usually hidden by prefixing it with #:
```suggestion
    /// # use hotshot_orchestrator::config::NetworkConfig;
```
This way it's still included when checking the doctest but isn't rendered in the docs.

Reply:
Woah, awesome! I had no idea that was possible. TIL!